### PR TITLE
Fix/microengine profits at min

### PR DESCRIPTION
**Pull request by ceseale**
Decreasing assertion fees for experts and adding tests to check BountyRegistry's final balance and that experts profit under minimum bounty conditions.

Related to https://github.com/polyswarm/development-private/issues/118

### CODE REVIEW

File: contracts/BountyRegistry.sol [hunk 1]
```diff
@@ -102,7 +102,7 @@ contract BountyRegistry is Pausable {
 
     // 0.0625NCT (1/16)
     uint256 public constant BOUNTY_FEE = 62500000000000000;
-    uint256 public constant ASSERTION_FEE = 62500000000000000;
+    uint256 public constant ASSERTION_FEE = 31250000000000000;
```
Comment (mjkoo) [inline contracts/BountyRegistry.sol]: Might want to edit or delete comment above

Comment (ceseale) [reply contracts/BountyRegistry.sol]: :+1:
